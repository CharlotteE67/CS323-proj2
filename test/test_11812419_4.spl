struct A{
    int a;
    float m;
    struct B{
        int c;
        char n;
    }b;
};

int main(){
    struct A sa;
    struct B sb;

    sa.a = sb.c;
    sb.c = sb.c + sb.n;
    sa.b = sb;

}