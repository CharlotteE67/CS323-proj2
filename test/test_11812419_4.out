Error type 7 at Line 15: unmatching operands.
Error type 5 at Line 16: unmatching types appear at both sides of the assigment operator.